Add const where we can

roystgnr opened the pull request:
I noticed these while debugging some polyhedra test failures

jwpeterson reviewed (approved):
These are non-reference parameters so I usually don't think about making such things `const`. Did this lead to you finding a bug or something?

moosebuild commented:
Job [Coverage](https://civet.inl.gov/job/2842893/), step Generate coverage on cad09f3 wanted to post the following:


## Coverage

<table>
  <thead>
    <tr>
      <th></th>
      <th>494ea3</th>
      <th colspan=4>#4158 cad09f</th>
    </tr>
    <tr>
      <th></th>
      <th>Total</th>
      <th>Total</th>
      <th>+/-</th>
      <th>New</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><b>Rate</b></td>
      <td>63.42%</td>
      <td>63.42%</td>
      <td>+0.00%</td>
      <td>100.00%</td>
    </tr>
    <tr>
      <td><b>Hits</b></td>
      <td>74837</td>
      <td>74842</td>
      <td>+5</td>
      <td>4</td>
    </tr>
    <tr>
      <td><b>Misses</b></td>
      <td>43167</td>
      <td>43162</td>
      <td>-5</td>
      <td>0</td>
    </tr>
  </tbody>
</table>

[Diff coverage report](https://mooseframework.inl.gov/libmesh/docs/PRs/4158/coverage_diff)

[Full coverage report](https://mooseframework.inl.gov/libmesh/docs/PRs/4158/coverage)



This comment will be updated on new commits.

roystgnr commented:
This is just the code I happened to be stepping through while I found an unrelated bug (now fixed) in the new polyhedra code.  Not seeing `const` where I expected it made me wonder whether there was something going on I wasn't understanding, so I added it to check.
